app/src/test/java/com/github/globant/githubsubscribers/subscribersdetail/SubscriberDetailPresenterTest.java

```diff
@@ -0,0 +1,225 @@
+    private static int following = 55;
+
+    /**
+     * Data for testing user repositories list.
+     */
+    private static String id_repo1 = "5284389";
+    private static String name_repo1 = "Crouton";
+    private static String fullName_repo1 = "keyboardsurfer/Crouton";
+    private static String html_url_repo1 = "https://github.com/keyboardsurfer/Crouton";
+
+    private static String id_repo2 = "10404007";
+    private static String name_repo2 = "homebrew-tap";
+    private static String fullName_repo2 = "keyboardsurfer/homebrew-tap";
+    private static String html_url_repo2 = "https://github.com/keyboardsurfer/homebrew-tap";
+
+    @Mock
+    private SubscriberDetailInteractor interactor;
+
+    @Mock
+    private SubscriberDetailView view;
+
+    @Captor
+    private ArgumentCaptor<SubscriberDetailInteractor.OnFinishedListener> OnFinishedListenerCallBack;
+
+    private SubscriberDetailPresenter presenter;
+
+    @Before
+    public void setupSubscriberDetailPresenter() {
+        MockitoAnnotations.initMocks(this);
+
+        // Get a reference to the class under test
+        presenter = new SubscriberDetailPresenterImpl(view, interactor);
+        // Setup mocking for Environment and File classes
+        mockStatic(Debug.class);
+    }
+
+    /**
+     * Unit Test to try when a successful answer is given from the User Info service
+     */
+    @Test
+    public void getUserInfoFromInteractorAndLoadIntoView() {
+
+        // Given an initialized User object with fake data
+        User user = new User(login, id, avatar_url, html_url, name, company, location, public_repos, followers, following);
+
+        // When detail presenter is called to get a User information
+        presenter.getUser(USER_NAME_PARAM);
+
+        //Then User information is loaded from Interactor and the callback is captured
+        verify(interactor).getUserData(eq(USER_NAME_PARAM), OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFinishedUser(user);
+
+        view.showSubscriberDetails(user);
+    }
+
+    /**
+     * Unit Test to try when a null answer is given from the User Info service
+     */
+    @Test
+    public void getNullUserInfoRequestFromInteractor() {
+
+        // Given a type error with fake error message
+        ErrorMessagesHelper.TypeError typeError = ErrorMessagesHelper.TypeError.BAD_ANSWER;
+        String errorMessage = Constants.EXCEPTION_ERROR + ": " + typeError.toString();
+
+        // When detail presenter is called to get a User information
+        presenter.getUser(USER_NAME_PARAM);
+
+        //Then User information is loaded from Interactor and the callback is captured
+        verify(interactor).getUserData(eq(USER_NAME_PARAM), OnFinishedListenerCallBack.capture());
+
+        OnFinishedListenerCallBack.getValue().onFailureUser(errorMessage, typeError);
+
+        int messageId = ErrorMessagesHelper.getMessage(typeError);
+        view.showUserError(messageId);
```

Comment:
Yo need to call `verify(view).method(param)`. Please refer to https://codelabs.developers.google.com/codelabs/android-testing/index.html?index=..%2F..%2Findex#5